subprojects/core/src/integTest/groovy/org/gradle/api/tasks/CopyPermissionsIntegrationTest.groovy

```diff
@@ -222,4 +223,34 @@ class CopyPermissionsIntegrationTest extends AbstractIntegrationSpec {
         cleanup:
         file('src/unauthorized').mode = 0777
     }
+
+    @Requires(TestPrecondition.FILE_PERMISSIONS)
+    @Issue('https://github.com/gradle/gradle/issues/9576')
+    def "unreadable excluded #type is ignored"() {
+        given:
+        def taskName = "withUnreadable$type"
+        def unreadable = create(file("targetFolder/unreadable$type"))
+        unreadable.readable = false
+
+        buildFile << """
+            task $taskName(type: Copy) {
+                from('.') { include '${buildFile.name}' }
+                into 'targetFolder'
+            }
+        """
+
+        when:
+        executer.withStackTraceChecksDisabled()
```

Comment:
Why is this needed?

Reply:
I was logging with `info`, now that I'm not checking the output and logging with `debug` I can remove it, thanks.

Reply:
Added it back, since I'm tracking debug logs now